code/modules/overmap/ships/computers/computer_shims.dm

```diff
@@ -94,7 +94,10 @@
 // If you don't call parent in this proc, you must make all appropriate checks yourself.
 // If you do, you must respect the return value.
 /obj/machinery/computer/ship/attack_hand(mob/user)
-	if((. = ..()))
+	if(..(user))
```

Comment:
Whai? It was already passing user upwards.  And now it won't return 1 on parent deny? Maybe it doesn't matter if nobody overrides this method, but whai?

Reply:
I was copying code from somewhere else and overwrote it in a hurry. Ill fix that.